docs/storagepolicies.md

```diff
@@ -144,19 +169,170 @@ metadata:
 
 ```
 apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
-  name: openebs-standard
+  name: openebs-jiva-default
   annotations:
     cas.openebs.io/config: |
       - name: VolumeMonitorImage
-        value: openebs/m-exporter:0.7.0-RC2
+        value: quay.io/openebs/m-exporter:v0.7.x-ci
+```
+
+### Volume Space Reclaim Policy
+
+Support for a storage policy that can disable the Jiva Volume Space reclaim.You can specify the jiva volume space reclaim feature setting using the *value* for *RetainReplicaData* property. In the following example, the jiva volume space reclaim feature is enabled as true. Hence, retain the volume data post PVC deletion.
+
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: openebs-jiva-default
+  annotations:
+    cas.openebs.io/config: |
+      - name: RetainReplicaData
+        enabled: true   
+```
+
+## Types of Storage Policies for cStor
+
+OpenEBS supports several types of Storage Policies for cStor volume such as the following.
+
+- cas.openebs.io/ReplicaCount
+- cas.openebs.io/VolumeControllerImage
+- cas.openebs.io/VolumeTargetImage
+- cas.openebs.io/StoragePoolClaim
+- cas.openebs.io/VolumeMonitor
+- cas.openebs.io/VolumeMonitorImage
+- cas.openebs.io/RetainReplicaData
+- cas.openebs.io/FSType
+
+### Replica Count Policy
+
+You can specify the cStor Pool replica count using the *value* for *ReplicaCount* property. In the following example, the ReplicaCount is specified as 3. Hence, three replicas for storage pool will be created.
+
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    cas.openebs.io/config: |
+      - name: ReplicaCount
+        value: "3"
+    openebs.io/cas-type: cstor
+```
+
+### Volume Controller Image Policy
+
+You can specify the cStor Volume Controller Image using the *value* for *VolumeControllerImage* property. This will help to choose the volume management image.
+
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    cas.openebs.io/config: |
+      - name: VolumeControllerImage
+        value: quay.io/openebs/cstor-volume-mgmt:v0.7.x-ci
+    openebs.io/cas-type: cstor
+```
+
+### Volume Target Image Policy
+
+You can specify the cStor Target Image using the *value* for *VolumeTargetImage* property. This will help to choose the cStor istgt target image.
+
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    cas.openebs.io/config: |
+      - name: VolumeTargetImage
+        value:quay.io/openebs/cstor-istgt:v0.7.x-ci
+    openebs.io/cas-type: cstor
+```
+
+### Storage Pool Claim Policy
+
+You can specify the cStor Pool Claim name using the *value* for *StoragePoolClaim* property. This will help to choose cStor storage pool name where OpenEBS volume will create. Following is the default StorageClass template where cStor volume will create on default cStor Sparse Pool.
```

Comment:
This will help you choose
OpenEBS volume will be created.
where cStor volume will be created on the default...